userguide/analytics/userguide_analytics.adoc

```diff
@@ -1,63 +1,595 @@
+
+A plugin can be installed via the https://github.com/killbill/killbill-cloud/blob/master/kpm[Kill Bill Package Manager (KPM)]. For this, please follow the steps given below:
+
+. Ensure that you have kpm installed as explained https://github.com/killbill/killbill-cloud/tree/master/kpm#kpm-installation[here].
+
+. Install the plugin using the following `kpm` command (Replace `<path_to_install_plugin>` with the path where you want to install the plugin. This path should match the path specified by the `org.killbill.osgi.bundle.install.dir` property in the  https://docs.killbill.io/latest/userguide_configuration.html#_configuration_properties_table[Kill Bill Configuration]):
+[source,bash]
+kpm install_java_plugin analytics --destination=<path_to_install_plugin>
+
+. Verify that the plugin is installed properly. This can be done by ensuring that the plugin JAR is present at the `path_to_install_plugin` specified in the previous step. Alternatively, you can also open https://docs.killbill.io/latest/userguide_kaui.html[Kaui], hover over the plug icon and click on `analytics`. This should display the following screen:
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/analytics-plugin/analytics-plugin-in-kaui.png[align=center]
+
+. If you would like to make changes to the plugin, you can clone the plugin code, build and deploy it as explained https://github.com/killbill/killbill-analytics-plugin[here].
+
+
+== Database Configuration
+
+As mentioned earlier, the analytics plugin uses its own reporting tables. To create these tables, please follow the steps given below:
+
+. Connect to the Kill Bill database.
+
+. Run the analytics plugin https://github.com/killbill/killbill-analytics-plugin/blob/0a25c18c3951b663e4d1ebf81619f325175209cf/src/main/resources/org/killbill/billing/plugin/analytics/ddl.sql[DDL].
+
+The <<analytics_tables, analytics tables>> section provides an overview of some of the important tables in the analytics database.
+
+== Plugin Configuration
+
+You can configure the following properties related to the analytics plugin in the https://docs.killbill.io/latest/userguide_configuration.html#global_configuration_properties[Kill Bill Configuration File]:
+
+[options="header",cols="1,1,1"]
+|===
+|Property Name   |Description   |Default Value   
+//----------------------
+|org.killbill.notificationq.analytics.tableName   |Specifies the name of the notification queue table   |notifications (Same as Kill Bill notification queue table)
+|org.killbill.notificationq.analytics.historyTableName   |Specifies the name of the notification queue history table    |notifications_history (Same as Kill Bill notification queue history table)
+|org.killbill.analytics.lockSleepMilliSeconds   |Specifies how long to sleep between retries to get the lock   |100   
+|org.killbill.billing.plugin.analytics.referenceCurrency   |Specifies the reference currency. See  <<currency_conversion,Currency Conversion>>  |USD   
+|===
+
```

Comment:
I suggest removing "Specifies" from the table above. I know the end result isn't a complete sentence, but it's preferable over needless repetition.

Reply:
Fixed